homeassistant/components/shelly/utils.py

```diff
@@ -298,7 +298,7 @@ def get_rpc_channel_name(device: RpcDevice, key: str) -> str:
         entity_name = device.config[key].get("name", device_name)
 
     if entity_name is None:
-        if [k for k in key if k.startswith(("input", "switch"))]:
+        if key.startswith(("input", "switch")):
```

Comment:
I guess its unlikely that we would ever get a key that was `input_bluetooth` or `switch_other` but it would be a bit safer it this was a split or the starts with included the `:` unless the goal is to match `input*`

Reply:
I will add the `:`, it means to take only the `input` or `switch` components